src/palette/internal/palette/palettecreator.cpp

```diff
@@ -1099,7 +1099,7 @@ PalettePanel* PaletteCreator::newBracketsPalettePanel()
             { BracketType::LINE,   QT_TRANSLATE_NOOP("palette", "Line") } }
     };
 
-    static Staff bracketItemOwner(nullptr);
+    static Staff bracketItemOwner(gscore);
```

Comment:
BTW I did try adding a staff to gscore if there wasn't one already and using that which avoids the need for another global variable here, but while it appeared to work OK, it's about 5 more lines of code and as it happens gscore is a global variable anyway, that's never freed (sure, the O/S will reclaim the memory for the MuseScore application, but what if another process repeatedly loads/unloads a dynamic link library that links to the engraving library?)

Reply:
The time will come soon. when we put it all in order :)